tests/test_units/test_table.py

```diff
@@ -111,3 +135,35 @@ def test_real_sum(edataobj3, drogon_summary):
     res = objdata._derive_objectdata()
     print("-----------------")
     assert res["table_index"] == ["DATE"], "Incorrect table index "
+
+
+def test_real_volumes(edataobj3, drogon_volumes):
+    objdata = _ObjectDataProvider(drogon_volumes, edataobj3)
+    res = objdata._derive_objectdata()
+    correct_answers = {
+        "ZONE": ["Valysar", "Therys", "Volon"],
+        "REGION": [
+            "WestLowland",
+            "CentralSouth",
+            "CentralNorth",
+            "NorthHorst",
+            "CentralRamp",
+            "CentralHorst",
+            "EastLowland",
+        ],
+        "FACIES": [
+            "Floodplain",
+            "Channel",
+            "Crevasse",
+            "Coal",
+            "Calcite",
+            "Offshore",
+            "Lowershoreface",
+            "Uppershoreface",
+        ],
+    }
+    print(res["table_index"])
```

Comment:
Forgotten `print` or intentional?

Reply:
Yes, have removed the print statements now